### PR TITLE
Fixes #27434 - icon_path instead origin to override

### PR DESCRIPTION
**Pull request by lzap**
Needs https://github.com/theforeman/foreman/pull/6585

### CODE REVIEW

File: app/models/katello/rhsm_fact_name.rb [hunk 1]
```diff
@@ -7,7 +7,11 @@ def set_name
     end
 
     def origin
-      'Redhat'
+      'RHSM'
+    end
```
Comment (lzap) [inline app/models/katello/rhsm_fact_name.rb]: This is shown on the mouse "hover" tooltip, unsure if this can beak anything else tho.

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
[test katello]

**Comment by lzap** [comment]
We are all set!

**Pull request merged**